mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
-## Unreleased
+## Unreleased (v1.18.1)
+
+* Update underlying S3 client version.
```

Comment:
Should we drop this with the next change to Mountpoint? I'm not sure we really want these entries in this change log.